server/utils/nunjucksSetup.ts

```diff
@@ -104,14 +106,17 @@ export default function nunjucksSetup(app: express.Express): void {
   njkEnv.addFilter('isOutside', isOutside)
   njkEnv.addFilter('isInPrison', isInPrison)
 
-  // utils for GDS & MoJ components
+  // application-provided utils for GDS & MoJ components
   njkEnv.addFilter('findFieldInGovukErrorSummary', findFieldInGovukErrorSummary)
   njkEnv.addFilter('govukCheckedItems', govukCheckedItems)
   njkEnv.addFilter('govukMultipleCheckedItems', govukMultipleCheckedItems)
   njkEnv.addFilter('govukCheckedItemsConditional', govukCheckedItemsConditional)
   njkEnv.addFilter('govukCheckedItemsDivider', govukCheckedItemsDivider)
   njkEnv.addFilter('govukSelectInsertDefault', govukSelectInsertDefault)
   njkEnv.addFilter('govukSelectSetSelected', govukSelectSetSelected)
+
+  // MoJ frontend components
+  njkEnv.addFilter('mojDate', mojFrontendFilters().mojDate)
```

Comment:
this is necessary for `mojTimeline` component to work even though no documentation mentions it